homeassistant/components/mastodon/strings.json

```diff
@@ -35,5 +35,18 @@
       "title": "YAML import failed with unknown error",
       "description": "Configuring {integration_title} using YAML is being removed but there was an unknown error while importing your existing configuration.\nPlease use the UI to configure Mastodon. Don't forget to delete the YAML configuration."
     }
+  },
+  "entity": {
+    "sensor": {
+      "followers": {
+        "name": "Followers"
+      },
+      "following": {
+        "name": "Following"
+      },
+      "statuses": {
+        "name": "Statuses"
+      }
```

Comment:
Is this what mastodon calls it?

Reply:
Changing to posts for both the key and translation name.
The API calls them statuses, but the JoinMastodon home page says "Mastodon supports audio, video and picture posts..."
They moved away from Toots in 2022.